scripts/deploy-function.sh

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env bash
+#     Copyright 2021 Dynatrace LLC
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+mkdir -p release
+cp activation-config.yaml "./release/activation-config.yaml"
+
+cd release/ || exit
+
+wget -q "https://github.com/dynatrace-oss/dynatrace-gcp-function/releases/download/${TRAVIS_TAG}/function-deployment-package.zip" -O function-deployment-package.zip; unzip function-deployment-package.zip; chmod a+x *.sh;
+
+cat <<EOF > activation.config.release.yaml
+googleCloud:
+  common:
+    dynatraceUrlSecretName: "${DYNATRACE_URL_SECRET_NAME}"
+    dynatraceAccessKeySecretName: "${DYNATRACE_ACCESS_KEY_SECRET_NAME}"
+    serviceAccount: "${IAM_SERVICE_ACCOUNT}"
+    iamRole: "${IAM_ROLE_METRCICS}"
+    requireValidCertificate: false
+  metrics:
+    pubSubTopic: "${PUBSUB_TOPIC}"
+    function: "${METRIC_FORWARDING_FUNCTION}"
+    scheduler: "${CLOUD_SCHEDULER}"
+activation:
+  metrics:
+    services:
+    # Google Cloud APIs
```

Comment:
should be changed to the new config: https://github.com/dynatrace-oss/dynatrace-gcp-function/blob/APM-310059-move-gcp-assets-to-ef/activation-config.yaml#L48